tests/web3js/eth_streaming_filters_test.js

```diff
@@ -65,7 +65,9 @@ async function assertFilterLogs(subscription, expectedLogs) {
 }
 
 it('streaming of logs using filters', async() => {
-    setTimeout(() => process.exit(1), (timeout-1)*1000) // hack if the ws connection is not closed
+    // hack if the ws connection is not closed
+    // TMP: disable
+    // setTimeout(() => process.exit(1), (timeout - 1) * 1000)
```

Comment:
I think with mocha set timeout we could remove this completely

Reply:
Good catch :+1: 
Removed in https://github.com/onflow/flow-evm-gateway/pull/321/commits/75279589ef60a3bc6d78f4f93405f858842ec99f